test/eth/ProxyCdp.spec.js

```diff
@@ -0,0 +1,299 @@
+import { buildTestEthereumCdpService } from '../helpers/serviceBuilders';
```

Comment:
if there's a subset of tests that you know we always want to run with both proxy and non-proxy cdp's, it might be worth de-duplicating the test code with something like this:

```js
const sharedTests = () => {
  test('1', () => {
    // ...
  });
  test('2', () => {
    // ...
  });
};

describe('non-proxy cdp', () => {
  beforeEach(() => {
    // non-proxy setup
  });
  describe('tests', sharedTests);
});

describe('proxy cdp', () => {
  beforeEach(() => {
    // proxy setup
  });
  describe('tests', sharedTests);
});
```